services/keybase/keybase-zec.tester.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const Joi = require('joi')
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('existing zcash address')
+  .get('/skyplabs.json')
+  .expectJSONTypes(
+    Joi.object({
+      name: 'zec',
+      value: Joi.string(),
```

Comment:
Again, `Joi.string()` will allow `not found`.

Reply:
Same as for Stellar addresses, I used the `withRegex(/^(?!not found$)/)` regex for now.